cfy_manager/status_reporter/status_reporter.py

```diff
@@ -0,0 +1,47 @@
+#########
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+from ..logger import get_logger
+from ..utils.systemd import systemd
+
+STATUS_REPORTER = 'statusreporter'
+
+logger = get_logger(STATUS_REPORTER)
+
+
+def configure():
+    logger.notice('Configuring component status reporting service with...')
+    # TODO: add all relevant inputs
+    # validate which input was entered and only update it
+    systemd.restart(STATUS_REPORTER)
+    logger.notice('Component status reporting service Configured...')
```

Comment:
We don't need an ellipsis following this, as that implies something is about to happen with regards to the message that was just logged.
Also, configured shouldn't be capitalised here.